core/modules/ckeditor5/ckeditor5.module

```diff
@@ -88,7 +88,7 @@ function ckeditor5_library_info() {
       $module_path . '/js/ckeditor5.formatter.js' => array(),
     ),
     'css' => array(
-      $module_path . '/css/ckeditor5.css' => array(),
+      $module_path . '/css/ckeditor5-backdrop.css' => array(),
```

Comment:
At first I thought we should avoid `backdrop` in the name of this CSS file. I recognize there's a conflict with `ckeditor5.css` in the build, so this name needs to change. But I couldn't think of a better name and although `backdrop` in CSS files is uncommon, we have many JS files with `backdrop` in them, including 5 files within the CKEditor 5 module. I think this might be an exceptional situation when we have a 3rd party library and are likely to encounter namespace conflicts.

So in short, I think this is fine.